test-utils/src/dev/test_cmds.rs

```diff
@@ -329,4 +322,19 @@ mod tests {
              path3: <VARIABLE_REDACTED>"
         );
     }
+
+    #[test]
+    fn test_redact_timestamps() {
+        let times = [DateTime::<Utc>::from_timestamp_nanos(0), Utc::now()];
```

Comment:
Somewhat ironically I worry about `Utc::now` triggering flakes later. Could we loop through a bunch of values for `from_timestamp_nanos` to test the varying subsecond lengths?

Reply:
Sure thing, updated with with a bunch of stable values.